src/Client.js

```diff
@@ -744,7 +744,9 @@ class Client extends EventEmitter {
      */
     async logout() {
         await this.pupPage.evaluate(() => {
-            return window.Store.AppState.logout();
+            if (window.Store && window.Store.AppState && typeof window.Store.AppState.logout === 'function') {
```

Comment:
Now it is safe and won't throw exception, but what is the root cause of the problem, why ```window.Store.AppState``` is ```undefined```?

Reply:
It's possible that the window.Store object or its AppState property is not fully initialized or defined at the time this code is executed. And if this code runs before the necessary initialization is complete, it can result in undefined variables.

Reply:
OK I understand :)

Reply:
I dont think its should do nothing if the client already destroyed 
Maybe throw specific expectation that said the client already destroyed.
Because after that function the use will think the logout success